components/rewards_data/BUILD.gn

```diff
@@ -0,0 +1,9 @@
+source_set("rewards_data") {
+  sources = [
+    "core/clear_rewards_data_tab.h",
```

Comment:
this stuff should go in `components/brave_rewards`

Reply:
Moved to `browser/rewards_data`